app/common/services/dataService.js

```diff
@@ -71,9 +71,11 @@ window.angular && (function(angular) {
       this.server_id = this.getServerId();
 
       this.setNetworkInfo = function(data) {
+        var formatted = data.formatted_data || {};
```

Comment:
Why do you need this var?

Reply:
This comment was cherry-pick from openbmc/[phosphor-webui](https://gerrit.openbmc.org/c/openbmc/phosphor-webui/+/36438).